php/utils.php

```diff
@@ -823,3 +823,16 @@ function parse_str_to_argv( $arguments ) {
 	}, $argv );
 	return $argv;
 }
+
+/**
+ * Locale-independent version of basename()
+ *
+ * @access public
+ *
+ * @param string $path
```

Comment:
Not sure what's your stance on this. Shouldn't we make sure all doc blocks contain complete information?
In this case, short summaries, with an `Optional.` and the default value for `$suffix` are needed.